apps/backend/src/groups/groups.service.ts

```diff
@@ -49,6 +51,18 @@ export class GroupsService {
     });
   }
 
+  async updateUserGroupRole(
+    group: Group,
+    updateGroupUser: AddUserToGroupDto
```

Comment:
Seems like there should be an `updateGroupUser` dto instead of repurposing addusertogroup